functions/EmuScripts/emuDeckCitra.sh

```diff
@@ -85,14 +86,14 @@ Citra_setupStorage(){
 #WipeSettings
 Citra_wipe(){
 	setMSG "Wiping $Citra_emuName config directory. (factory reset)"
-	rm -rf "$HOME/.var/app/$Citra_emuPath"
+	rm -rf "$HOME/.config/citra-emu"
 }
 
 
 #Uninstall
 Citra_uninstall(){
 	setMSG "Uninstalling $Citra_emuName."
-    flatpak uninstall "$Citra_emuPath" --user -y
+    cd ./citra && ./maintenancetool purge
```

Comment:
this assumes a starting path. I'd make sure it's the complete path to the binaries.